bootstrapper/internal/journald/journald.go

```diff
@@ -31,17 +32,31 @@ func NewCollector(ctx context.Context) (*Collector, error) {
 	if cmd.Err != nil {
 		return nil, cmd.Err
 	}
-	pipe, err := cmd.StdoutPipe()
+	stdoutPipe, err := cmd.StdoutPipe()
 	if err != nil {
 		return nil, err
 	}
-	return &Collector{cmd, pipe}, nil
+	stderrPipe, err := cmd.StderrPipe()
+	if err != nil {
+		return nil, err
+	}
+	collector := Collector{cmd, stdoutPipe, stderrPipe}
+	return &collector, nil
 }
 
 // Pipe returns a pipe to read the systemd logs. This should be read with a bufio Reader.
 func (c *Collector) Pipe() (io.ReadCloser, error) {
 	if err := c.cmd.Start(); err != nil {
 		return nil, err
 	}
-	return c.pipe, nil
+	return c.stdoutPipe, nil
+}
+
+// Stderr returns the error message of the journalctl command.
+func (c *Collector) Stderr() ([]byte, error) {
```

Comment:
This still doesn't check the exit code. We should wait for the command to finish, then check the error of the command, then return this error and stderr.